### ISSUE TITLE
make cache encoding optional for shopping cart

### ISSUE THREAD

**Issue by michael-conway** [DICE-UNC/jargon#365]
allow shopping cart entries in data cache to be plain text by making null/empty key possible

**Comment by korydraughn** [comment]
@michael-conway I see a commit against this issue. Is the work for this issue complete?

**Comment by trel** [comment]
shopping cart should not be a jargon concept?

**Comment by korydraughn** [comment]
I agree, but it appears to have already been merged into the main branch.

If it has been merged, we should pull it out and put it into a separate repo.

Was this feature added specifically for metalnx?

**Comment by trel** [comment]
I believe so.   Multiple selection, before 'bundling' and then download.

**Comment by korydraughn** [comment]
In that case, the shopping cart functionality should be moved directly into metalnx.
